README.md

```diff
@@ -1,2 +1,99 @@
 # open-sir
-Python SIR-x model implementation
+
+Open-SIR is an Open Source Python project for modelling pandemics and infectious diseases using Compartmental Models, such as the widely used [Susceptible-Infected-Removed (SIR) model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics). 
+
+The current stage of the software is *Alpha*.
+
+## Features
+- Model the dynamics of infectious diseases
+- Parameter fitting
+- Calculation of confidence intervals
+- CLI for interfacing with non Python environments (Bash, Node.JS, Matlab, etc).
+
+So far, Open-SIR provides an implementation of the SIR model and the recently new SIR based model SIR-X, developed by the [Robert Koch Institut](http://rocs.hu-berlin.de/corona/docs/forecast/model/#sir-x-dynamics-outbreaks-with-temporally-increasing-interventions).
+
+## Getting Started
+
+Open-SIR uses [Pipenv](https://pipenv.pypa.io/en/latest/) to automatically create a virtual environment and manage python packages. The python packages required by Open-SIR are listed in the [Pipfile](Pipfile).
+
+### Dependencies
+* Python 3.7
+* Pipenv
+* NumPy
+* SciPy
+* Jupyter
+* Matplotlib
+
+### Installing (for developers now)
+
+After cloning the repository, change the current directory to the repository by `cd open-sir` and automatically install the environment from the Pipfile using Pipenv:
+
+```
+pipenv install
+```
+
+Next, activate the Pipenv shell:
+```
+pipenv shell
+```
+
+In the Pipenv shell, check that the installation was successful calling the CLI open-sir.py
+
+```
+python open-sir.py -p '[0.95,0.38]' -i '[341555,445,0]' -t 6 
+```
+
+Alternatively, run the same command through Pipenv outside the open-sir Pipenv environment:
```

Comment:
```suggestion
Alternatively, instead of activating the shell, you can run the same command through Pipenv outside the open-sir Pipenv environment:
```